sdks/python/container/boot.go

```diff
@@ -73,13 +74,20 @@ const (
 )
 
 func main() {
+	if err := mainError(); err != nil {
+		log.Print(err)
+		os.Exit(1)
+	}
+}
+
+func mainError() error {
 	flag.Parse()
```

Comment:
Conventionally, flag.Parse() should remain in main(), rather some called out function.

Reply:
This should still be in the actual main(), and not tucked into the function.

Reply:
Done.